app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -218,9 +213,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                         cropCapturedImage(selectedImageUri);
                     } catch (ActivityNotFoundException aNFE) {
                         //display an error message if user device doesn't support
-                        String errorMessage = "Sorry - your device doesn't support the crop action!";
-                        Toast toast = Toast.makeText(this, errorMessage, Toast.LENGTH_SHORT);
-                        toast.show();
+                        showToast(getString(R.string.error_crop_not_supported));
```

Comment:
Yeah, I know.
I was just specifying that these are the type of message we need and not the earlier one.